easybuild/easyconfigs/x/xrandr/xrandr-1.4.3-intel-2015a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'xrandr'
+version = '1.4.3'
+
+description = "xrandr-1.3.5: Primitive command line interface to RandR extension"
+homepage = 'http://xorg.freedesktop.org/'
+
+sources = ['xrandr-1.3.5.tar.bz2']
+source_urls = ['http://pkgs.fedoraproject.org/repo/pkgs/xorg-x11-server-utils/xrandr-1.3.5.tar.bz2/9735173a84dca9b05e06fd4686196b07/']
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+dependencies = [
+    ('xproto', '7.0.27'),
+    ('libXrender', '0.9.8'),
+    ('libX11', '1.6.2', '-Python-2.7.9'),
+    ('libXrandr', '1.4.2')
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['.'] # fixme when the compilation is successful
+}
+
+parallel = 1 # this is a conservative choice
+maxparallel = 1 # this is a conservative choice
```

Comment:
only one makes sense, I'd prefer `parallel = 1`, since `maxparallel` is intended for situations where parallel building *does* work, but only up to a certain level